Dangerfile

```diff
@@ -1,46 +1,31 @@
 # frozen_string_literal: true
 
-def wip_feature?
-  has_wip_label = github.pr_labels.any? { |label| label.include?('WIP') }
-  has_wip_title = github.pr_title.include?('WIP')
+github.dismiss_out_of_range_messages
 
-  has_wip_label || has_wip_title
-end
+# `files: []` forces rubocop to scan all files, not just the ones modified in the PR
+rubocop.lint(files: [], force_exclusion: true, inline_comment: true, fail_on_inline_comment: true, include_cop_names: true)
 
-# these are reviewers actively providing feedback and potentially changing the state of the PR (approved, changes-requested)
-def active_reviewers?
-  repo_name = github.pr_json['base']['repo']['full_name']
-  pr_number = github.pr_json['number']
+manifest_pr_checker.check_all_manifest_lock_updated
 
-  !github.api.pull_request_reviews(repo_name, pr_number).empty?
+# skip remaining checks if we're during the release process
+if github.pr_labels.include?('Releases')
+  message('This PR has the `Releases` label: some checks will be skipped.')
+  return
 end
 
-# requested_teams / requested_reviewers are users initially requested to review a PR, who haven't reacted yet
-def requested_reviewers?
-  has_requested_reviews = !github.pr_json['requested_teams'].to_a.empty? || !github.pr_json['requested_reviewers'].to_a.empty?
-  has_requested_reviews || active_reviewers?
-end
+view_changes_checker.check
 
-return if github.pr_labels.include?('Releases')
+pr_size_checker.check_diff_size(max_size: 500)
```

Comment:
Should we log a note about skipped checks like we do for release PRs?

The opposite suggestion would also apply. If we don't log a message for drafts, we might want to avoid it for release PRs, too. 

I can't put my finger on why, but I feel like not logging a message for release PRs would be inappropriate—maybe it's just that I'm used to see it there, though. Under that rationale, I'd like to have consistency and log a message here, too, but it's not something I feel strongly about.

Reply:
My rationale for having it in release mode but not on Drafts is mainly due to the scope of what's being skipped: most of the checks are skipped if we are in the release process, so it makes sense to make it clear. Just a couple of more superficial checks (e.g. labels) are skipped for Drafts, to avoid unnecessary noise while the PR creator just wants to try some things out.

I understand the question though. I think we can log it just incase, and if we feel it's unnecessary we can remove it later.